lib/codelog/command/new.rb

```diff
@@ -5,19 +5,24 @@ module Command
     class New
       include FileUtils
 
-      def self.run(options)
-        Codelog::Command::New.new.run options
+      def initialize(name, options)
+        @name = name
+        @options = options
       end
 
-      def run(options)
+      def self.run(name, options)
+        Codelog::Command::New.new(name, options).run
```

Comment:
Did you consider using the 
```ruby
def self.run(name, options)
  new(name, options).run
end
```
format?

Reply:
My bad. I'll fix it

Reply:
Your approach is not wrong, its just different

Reply:
It's not wrong, but I think it is better to let the class name out